External/FEXCore/Source/Interface/IR/Passes.h

```diff
@@ -11,7 +11,7 @@ class Pass;
 class RegisterAllocationPass;
 class RegisterAllocationData;
 
-std::unique_ptr<FEXCore::IR::Pass> CreateConstProp(bool InlineConstants);
+std::unique_ptr<FEXCore::IR::Pass> CreateConstProp(bool InlineConstants, bool SupportsRCPC2);
```

Comment:
I'd rather us name this with a more portable name, like 'SupportsTSOImm9'

Reply:
Changed over to `SupportsTSOImm9`. Resolved.